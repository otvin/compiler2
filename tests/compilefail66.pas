program compilefail66(output);
{invalid return type for a function}


type q = array[1..5] of boolean;
var myq:q;

function x(a:integer):q;
begin
	x[a] := true;
end;

begin
	myq := x(1);
end.
