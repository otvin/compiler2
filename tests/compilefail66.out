Function x has invalid return type: q at file:tests/compilefail66.pas line:8     column:1  
